dev/PushNotifications/PushNotificationUtility.h

```diff
@@ -135,4 +143,135 @@ namespace winrt::Microsoft::Windows::PushNotifications::Helpers
     {
         return AppModel::Identity::IsPackagedProcess() && IsBackgroundTaskBuilderAvailable();
     }
+
+    inline HRESULT GetPackageFullName(wil::unique_cotaskmem_string& packagedFullName) noexcept try
+    {
+        WCHAR packageFullName[PACKAGE_FULL_NAME_MAX_LENGTH + 1]{};
+        UINT32 packageFullNameLength{ ARRAYSIZE(packageFullName) };
+        THROW_IF_FAILED(GetCurrentPackageFullName(&packageFullNameLength, packageFullName));
+
+        packagedFullName = wil::make_cotaskmem_string(packageFullName);
+        THROW_IF_NULL_ALLOC(packagedFullName);
+
+        return S_OK;
+    }
+    CATCH_RETURN()
+
+    inline HRESULT GetExeNameFromPath(std::wstring& exePath) noexcept try
```

Comment:
This is a weird API for 2 reasons: It takes an already existing construct in a path format as an argument and modifies it to a pure filename. Its a bad pattern.

Instead try
std::string GetExeNameFromPath(std::wstring& const exepath)